mecademic_description/urdf/meca_500_r3_macro.xacro

```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://wiki.ros.org/xacro">
+
+  <xacro:macro name="meca_500_r3_robot" params="
+    prefix
+    robot_ip
+    control_port
+    monitoring_port
+    connected_to:=''
+    velocity_timeout:=0.1
+    velocity_limit_percent:=25.0
+    acceleration_limit_percent:=100.0
+    use_fake_hardware:=false
+    fake_sensor_commands:=false
+    initial_positions:=${dict(axis_1_joint=0.0,axis_2_joint=0.0,axis_3_joint=0.0,axis_4_joint=0.0,axis_5_joint=0.0,axis_6_joint=0.0)}">
+
+    <!-- ros2 control include -->
+    <xacro:include filename="$(find mecademic_description)/urdf/meca_500_r3.ros2_control.xacro" />
+    <!-- ros2 control instance -->
+    <xacro:meca_500_r3_ros2_control
```

Comment:
We used to seperate the ros2 control macro from the robot macro, and then have an urdf argument that decides if we include the ros2 control tags. Did you do it this way on purpose?